web/src/user_topic_popover.ts

```diff
@@ -8,7 +9,10 @@ import {parse_html} from "./ui_util";
 import * as user_topics from "./user_topics";
 import * as util from "./util";
 
-export function initialize() {
+let stream_id: number | undefined;
+let topic_name: string | undefined;
```

Comment:
Why do we need to introduce these global variables? I feel like the prior code's approach of parsing these values from the DOM was a nicer set of abstractions.

Reply:
Hi @timabbott  , thank you so much for your review! I agree with you, they are actually unnecessary... I have removed them now 😊